### ISSUE TITLE
[package] libcurl: cacert.pem updated, sha256 check failing

### ISSUE THREAD

**Issue by dpaulat** [conan-io/conan-center-index#15196]
### Description

The cacert.pem at https://curl.haxx.se/ca/cacert.pem was updated, and the installation of the package is failing:

```
libcurl/7.87.0: Configuring sources in D:\.conan\data\libcurl\7.87.0\_\_\source\src
Downloading curl-7.87.0.tar.gz completed [4190.55k]                                      libcurl/7.87.0: rl/7.87.0:
libcurl/7.87.0:
Downloading cacert.pem completed [211.51k]                                               libcurl/7.87.0:
libcurl/7.87.0:
ERROR: libcurl/7.87.0: Error in source() method, line 229
        download(self, "https://curl.haxx.se/ca/cacert.pem", "cacert.pem", verify=True, sha256="2cff03f9efdaf52626bd1b451d700605dc1ea000c5da56bd0fc59f8f43071040")
        ConanException: sha256 signature failed for 'cacert.pem' file.
 Provided signature: 2cff03f9efdaf52626bd1b451d700605dc1ea000c5da56bd0fc59f8f43071040
 Computed signature: fb1ecd641d0a02c01bc9036d513cb658bbda62a75e246bedbc01764560a639f0
```

Assuming the certificate update is legitimate, the recipe likely needs updated. This impacts multiple (all?) versions of the package.

### Package and Environment Details

* Package Name/Version: libcurl/7.87.0
* Operating System+version: Windows 11 22H2 (22621.963)
* Compiler+version: MSVC 2022 (17.3.1)
* Conan version: conan 1.56.0
* Python version: Python 3.10.3

### Conan profile

Configuration for profile default:

[settings]
os=Windows
os_build=Windows
arch=x86_64
arch_build=x86_64
compiler=Visual Studio
compiler.version=17
build_type=Release
[options]
[conf]
[build_requires]
[env]
conan install -r conancenter libcurl/7.87.0@ -pr:b=default -pr:h=default

### Steps to reproduce

conan install -r conancenter libcurl/7.87.0@ --build=libcurl

### Logs

<details><summary>Click to expand log</summary>

```
>conan install -r conancenter libcurl/7.87.0@ --build=libcurl
Configuration:
[settings]
arch=x86_64
arch_build=x86_64
build_type=Release
compiler=Visual Studio
compiler.runtime=MD
compiler.version=17
os=Windows
os_build=Windows
[options]
[build_requires]
[env]

libcurl/7.87.0: Forced build from source
Installing package: libcurl/7.87.0
Requirements
    libcurl/7.87.0 from 'conancenter' - Cache
    openssl/1.1.1s from 'conancenter' - Cache
    zlib/1.2.13 from 'conancenter' - Cache
Packages
    libcurl/7.87.0:2517a780d01cde682aedd711f5cddf33f09d6734 - Build
    openssl/1.1.1s:5a61a86bb3e07ce4262c80e1510f9c05e9b6d48b - Cache
    zlib/1.2.13:5a61a86bb3e07ce4262c80e1510f9c05e9b6d48b - Cache

Installing (downloading, building) binaries...
openssl/1.1.1s: Already installed!
zlib/1.2.13: Already installed!
libcurl/7.87.0: WARN: Trying to remove corrupted source folder
libcurl/7.87.0: WARN: This can take a while for big packages
libcurl/7.87.0: Configuring sources in D:\.conan\data\libcurl\7.87.0\_\_\source\src
Downloading curl-7.87.0.tar.gz completed [4190.55k]                                      libcurl/7.87.0: rl/7.87.0:
libcurl/7.87.0:
Downloading cacert.pem completed [211.51k]                                               libcurl/7.87.0:
libcurl/7.87.0:
ERROR: libcurl/7.87.0: Error in source() method, line 229
        download(self, "https://curl.haxx.se/ca/cacert.pem", "cacert.pem", verify=True, sha256="2cff03f9efdaf52626bd1b451d700605dc1ea000c5da56bd0fc59f8f43071040")
        ConanException: sha256 signature failed for 'cacert.pem' file.
 Provided signature: 2cff03f9efdaf52626bd1b451d700605dc1ea000c5da56bd0fc59f8f43071040
 Computed signature: fb1ecd641d0a02c01bc9036d513cb658bbda62a75e246bedbc01764560a639f0
```

</details>

**Comment by mkoviazin** [comment]
#15195 should fix that, so I'll close my PR

**Comment by dpaulat** [comment]
Fixed by #15195
